src/Cryptography/Sodium/Bindings/Scrypt.hs

```diff
@@ -0,0 +1,253 @@
+{-# LANGUAGE CApiFFI #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE Trustworthy #-}
+
+-- |
+--
+-- Module: Cryptography.Sodium.Bindings.Scrypt
+-- Description: Direct bindings to the scrypt password hashing function.
+-- Copyright: (C) Hécate Moonlight 2022
+-- License: BSD-3-Clause
+-- Maintainer: The Haskell Cryptography Group
+-- Stability: Stable
+-- Portability: GHC only
+module Cryptography.Sodium.Bindings.Scrypt
+  ( -- * Introduction
+    -- $introduction
+
+    -- * Key Derivation
+    cryptoPWHashScryptSalsa2018SHA256,
+
+    -- * Password storage
+    cryptoPWHashScryptSalsa2018SHA256Str,
+    cryptoPWHashScryptSalsa2018SHA256StrVerify,
+
+    -- * Constants
+    cryptoPWHashScryptSalsa2018SHA256BytesMin,
+    cryptoPWHashScryptSalsa2018SHA256BytesMax,
+    cryptoPWHashScryptSalsa2018SHA256PasswdMin,
+    cryptoPWHashScryptSalsa2018SHA256PasswdMax,
+    cryptoPWHashScryptSalsa2018SHA256SaltBytes,
+    cryptoPWHashScryptSalsa2018SHA256StrBytes,
+    cryptoPWHashScryptSalsa2018SHA256StrPrefix,
+    cryptoPWHashScryptSalsa2018SHA256OpsLimitMin,
+    cryptoPWHashScryptSalsa2018SHA256OpsLimitMax,
+    cryptoPWHashScryptSalsa2018SHA256MemLimitMin,
+    cryptoPWHashScryptSalsa2018SHA256MemLimitMax,
+    cryptoPWHashScryptSalsa2018SHA256OpsLimitInteractive,
+    cryptoPWHashScryptSalsa2018SHA256MemLimitInteractive,
+    cryptoPWHashScryptSalsa2018SHA256OpsLimitSensitive,
+    cryptoPWHashScryptSalsa2018SHA256MemLimitSensitive,
+  )
+where
+
+import Foreign (Ptr)
+import Foreign.C (CChar (..), CInt (CInt), CSize (CSize), CUChar, CULLong (CULLong))
+
+-- $introduction
+-- This is an implementation of the scrypt password hashing function.
+-- However, unless you have specific reasons to use scrypt, you __should__ instead consider the default function Argon2.
```

Comment:
Isn't emphasis in Haddock `/like this/`?

Reply:
Yes, but here I'm doing **bold**